src/cljs/rems/application.cljs

```diff
@@ -387,7 +385,7 @@
      (application-list/format-application-id config application)]))
 
 (defn- format-event [event]
-  {:user (:commonName (:event/actor-attributes event))
+  {:user (:name (:event/actor-attributes event)) ;; TODO use get-member-name?
```

Comment:
Yeah, sorry, this is my bad, please change to get-member-name. I changed this recently to access :commonName there, but it should have been get-member-name all along.

Reply:
done